.bazelrc

```diff
@@ -167,7 +167,7 @@ build:linux --config=dynamic_kernels
 # the monolitih build because otherwise there are missing dependencies and
 # linking fails.
 build --define framework_shared_object=false
-build --define tsl_protobuf_header_only=true
+build --define tsl_protobuf_header_only=false
```

Comment:
@ManfeiBai do we know why this change resolves the recent failures?

Reply:
the recent failures is due to missing proto after delete `core:lib`, and `--define tsl_protobuf_header_only=false` will enable PyTorch/XLA build with these protos again